### DISCUSSION TITLE
Typo in reload-live-server package.

### DISCUSSION THREAD

**Discussion by AlexanderStoilov** [npm/feedback#1085] [Website] [answered]
Hi guys, there seems to be a typo in the instructions for installation via **npm** - `reload-reload-...`.
Screenshot: ![image](https://github.com/npm/feedback/assets/79380589/e5d34e49-32dd-4e51-a737-7892681c85d2)

**Comment by MylesBorins** [comment, marked as answer]
@AlexanderStoilov this feedback repo is intended to provide feedback about the npm registry itself. It seems like you need to direct this feedback to the maintainers of this specific package.
